wiki/Tournaments/MCNC/2021/en.md

```diff
@@ -0,0 +1,130 @@
+| Finals - Week 1 | 2021-05-24/2021-05-30 |
+| Finals - Week 2 | 2021-05-31/2021-06-06 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | CNY 2,500, unique profile badge, 4 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | CNY 1,500, 2 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | CNY 800, 1 months of osu!supporter tag |
+
+## Organisation
+
+The osu!mania 4K Chinese National Cup 2021 was run by various community members.
+
+| Position | Members |
+| :-- | :-- |
+| Manager | ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832), ![][flag_US] [dblade](https://osu.ppy.sh/users/4272841) |
+| Map selector | ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832), ![][flag_CN] [\[Illustrious\]](https://osu.ppy.sh/users/4082595), ![][flag_CN] [Muses](https://osu.ppy.sh/users/9705896), ![][flag_CN] [\[Crz\]MemoryI](https://osu.ppy.sh/users/8179131), ![][flag_CN] [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), ![][flag_MX] [\[Crz\]Noire](https://osu.ppy.sh/users/6031847), ![][flag_MX] [\[OSC\]Amagai](https://osu.ppy.sh/users/9658070) |
+| Streamer | ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832), ![][flag_CN] [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), ![][flag_CN] [\[Illustrious\]](https://osu.ppy.sh/users/4082595), ![][flag_CN] [Sonoaoi](https://osu.ppy.sh/users/9755808), ![][flag_CN] [110](https://osu.ppy.sh/users/4524921), ![][flag_CN] [\[Crz\]Mix0130](https://osu.ppy.sh/users/7215250) |
+| Commentator | ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832), ![][flag_CN] [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), ![][flag_CN] [\[Illustrious\]](https://osu.ppy.sh/users/4082595), ![][flag_CN] [\[Crz\]Mix0130](https://osu.ppy.sh/users/7215250) |
+| Designer | ![][flag_CN] [Dr\_Tissues](https://osu.ppy.sh/users/5106681) |
+| Referee | ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832), ![][flag_CN] [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), ![][flag_CN] [\[Illustrious\]](https://osu.ppy.sh/users/4082595), ![][flag_CN] [\[Crz\]MemoryI](https://osu.ppy.sh/users/8179131), ![][flag_CN] [Ayase Eli](https://osu.ppy.sh/users/6764156) |
+| Statistician | ![][flag_CN] [Ayase Eli](https://osu.ppy.sh/users/6764156), ![][flag_CN] [\[Crz\]xz1z1z](https://osu.ppy.sh/users/10500832) |
+
+## Links
+
+- [Discussion thread (English)](https://osu.ppy.sh/community/forums/topics/1260186)
+- [Discussion thread (Chinese)](https://osu.ppy.sh/community/forums/topics/1260175)
+- [Livestream](https://live.bilibili.com/2996250)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1xgxKNKoJNEQz_zlPUf-4fyxeBDpuhfkgqxXyOxaIkpU)**
+
+## Mappools
+
+### Qualifiers
+
+**[Download the mappack here! (18 MB)](https://drive.google.com/file/d/1KYzdeiVoa11U0FYwjzqu-EKnDiA8DFr4/view?usp=sharing)**
+
+- NoMod
```

Comment:
the maps in the mappool should be written in this order actually : `Artist - Title (Mapset Host) [Difficulty]`

![image](https://user-images.githubusercontent.com/36564236/113840873-583cf300-97bb-11eb-8874-ed981d6c2716.png)

so in your case you should write your pool as follows :

```- [Petit Rabbit's - Tenkuu Cafeteria (TV Size) (Hidden is fun) \[BLOOM! (No LN)\]](https://osu.ppy.sh/b/2923231)```
```- [Makoto (CV: Komatsu Mikako) / Kasumi (CV: Minase Inori) - Mikaiketsuna omoi (\[Crz\]Satori) \[ln\]](https://osu.ppy.sh/b/2924900)```
```(and so on)```

Reply:
also are you sure these are all NoMod instead of FreeMod? it's very unconventional for an osu!mania tournament to have a forced NoMod pool tho O_O

Reply:
Done!